apps/hash-graph/bin/cli/src/subcommand/server.rs

```diff
@@ -314,3 +328,20 @@ pub async fn server(args: ServerArgs) -> Result<(), GraphError> {
 
     Ok(())
 }
+
+pub async fn healthcheck(address: ApiAddress) -> Result<(), GraphError> {
+    let request_url = format!(
+        "http://{}:{}/api-doc/openapi.json",
+        address.api_host, address.api_port
+    );
+
+    Client::new()
+        .head(&request_url)
```

Comment:
I suppose we do `head` because we don't really want to fetch the data but check for responsiveness

Reply:
Yes, exactly 🙂